code/modules/mob/living/carbon/carbon.dm

```diff
@@ -475,6 +475,12 @@
 	if(stat == UNCONSCIOUS)
 		blind_eyes(1)
 		disabilities |= DEAF
+		if(ishuman(src)) //registers signal for xeno critdragging
+			var/mob/living/carbon/human/human = src
+			RegisterSignal(human, COMSIG_MOVABLE_PULL_MOVED, /mob/living/carbon/human.proc/oncritdrag)
```

Comment:
1) why is it here and not when the trait knocked out is added / removed?
2) don't istype(src), you're supposed to know what type src is in a proc of src.